src/main/java/se/sundsvall/supportmanagement/service/mapper/CommunicationMapper.java

```diff
@@ -179,9 +180,9 @@ private CommunicationAttachmentEntity toMessageAttachment(final String namespace
 			.withMunicipalityId(municipalityId)
 			.withId(UUID.randomUUID().toString())
```

Comment:
Why are we doing this?

Why not add `@UuidGenerator` on the id-field in the CommunicationAttachmentEntity-class?